MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -4344,10 +4304,16 @@
     {
       "Id": "ChestWoodfallBlueRupee",
       "RequiredItems": [
-        "AreaSouthAccess",
-        "Woodfall Platforming"
```

Comment:
Why make this change? These are equivalent.

Reply:
I deleted Woodfall Platforming because all the checks in Woodfall have different requirements, and because I added long jumping as FD to get to Woodfall without needing the jump to the Soaring Tablet which is defined as a FD Difficult Jump.

Reply:
But Woodfall Platforming still exists. Were you supposed to remove it and update the entries that use it?

Reply:
I did realize that I forgot to delete it, and also forgot to update the 2 pots in Woodfall. It's fixed in the most recent revision.